### PR TITLE
Fix integration tests sometimes failing

### PR DESCRIPTION
**Pull request by noituri**
- ~Added `LIVE_COMPOSITOR_NEVER_DROP_OUTPUT_FRAMES` env~
- Made output receiver dump longer
- Lowered resolution
- Changed `encoder_preset` to `ultrafast`

Snapshots PR: https://github.com/membraneframework-labs/video_compositor_snapshot_tests/pull/30

### CODE REVIEW

File: compositor_integration_tests/src/compositor_instance.rs [hunk 1]
```diff
@@ -91,6 +91,7 @@ fn get_free_port() -> u16 {
 fn init_compositor_prerequisites() {
     static GLOBAL_PREREQUISITES_INITIALIZED: OnceLock<()> = OnceLock::new();
     GLOBAL_PREREQUISITES_INITIALIZED.get_or_init(|| {
+        env::set_var("LIVE_COMPOSITOR_NEVER_DROP_OUTPUT_FRAMES", "1");
```
Comment (wkozyra95) [inline compositor_integration_tests/src/compositor_instance.rs:94]: This should not be necessary if you lowered resolution. Is the issue still happening?

File: snapshot_tests/snapshots
Comment (wkozyra95) [inline snapshot_tests/snapshots:1]: What about input streams shouldn't we use lower resolution for them too?

**Review by wkozyra95 — APPROVED** [review]
This is fine, but you are mixing different things without any good reason. In my opinion proper approach here would be to either:
- Only add LIVE_COMPOSITOR_NEVER_DROP_OUTPUT_FRAMES in this PR. Fix issue properly in follow PR
- Fix issue in first PR and do not add LIVE_COMPOSITOR_NEVER_DROP_OUTPUT_FRAMES at all

While this PR adds to both temporary hack and half of the proper solution.

File: snapshot_tests/snapshots
Comment (noituri) [reply snapshot_tests/snapshots:1]: Input streams are already low resolutions (640x360). It shouldn't be a problem I think

File: compositor_integration_tests/src/compositor_instance.rs (hunk 1 above)
Comment (noituri) [reply compositor_integration_tests/src/compositor_instance.rs]: I didn't test without it. But it seems to work without that env

**Pull request merged**